tooling/minimize/tests/pass/ops.rs

```diff
@@ -5,12 +5,17 @@ fn black_box<T>(t: T) -> T { t }
 
 fn main() {
     print(-black_box(-42));
+    print(!black_box(-43));
```

Comment:
Please also check an unsigned number, like `!1u32` becoming `(u32::MAX-1)`

Reply:
Added `assert!(!black_box(i32::MIN) == i32::MAX);` as well for good measure :)